src/ontology/imports/uo-extracted.owl

```diff
@@ -30,7 +30,7 @@
     <!-- http://purl.obolibrary.org/obo/IAO_0000115 -->
 
     <owl:AnnotationProperty rdf:about="http://purl.obolibrary.org/obo/IAO_0000115">
-        <rdfs:label>definition</rdfs:label>
+        <rdfs:isDefinedBy rdf:resource="http://purl.obolibrary.org/obo/uo.owl"/>
```

Comment:
Is this change intended? It deletes the label of the annotation property `definition`.
This this related to the error in the build script. As far as I understand the script, it uses uses the labels:
> current_df = df[df["LABEL"].str.lower().str.startswith(letter)]
But then fails due to this missing label.

Reply:
Yes, this is actuall intended. I removed it to recuce the number of "label" annotations:
![grafik](https://github.com/OpenEnergyPlatform/ontology/assets/38690039/24c3cc8c-80c0-44f9-b7eb-a1da4a1f7428)